cmake/FindGSL.cmake

```diff
@@ -118,7 +118,7 @@ ELSE(WIN32)
         )
       
       ELSE(GSL_CONFIG)
-        MESSAGE("FindGSL.cmake: gsl-config not found. Please set it manually. GSL_CONFIG=${GSL_CONFIG}")
+        MESSAGE(FATAL_ERROR "FindGSL.cmake: gsl-config not found. Please set it manually. GSL_CONFIG=${GSL_CONFIG}")
```

Comment:
Could this be made a bit more verbose? And what I also wonder is the use of `${GSL_CONFIG}` in here, after all it's in the else branch where it has not been found.

> FindGSL.cmake: gsl-config not found. Please install the `libgsl` development libraries or set the path with `-DGSL_CONFIG=/path/to/gsl-config`.

(I know it wasn't you who wrote it ;) )